internal/test/issues/issue-1378/common/config.yaml

```diff
@@ -0,0 +1,9 @@
+package: common
+generate:
+  gorilla-server: true
+  embedded-spec: true
+  strict-server: true
+  models: true
+output-options:
```

Comment:
Probably not for this PR, but it would make sense to not prune code that is used in dependent APIs by default, instead of having to set this explicitly.

Reply:
Raised #1442 to cover this